### DISCUSSION TITLE
Smoke works with the "No Scope" Mod enabled.

### DISCUSSION THREAD

**Discussion by Polyva** [ppy/osu#20731] [Ideas] [answered]
I think Smoke should get disabled when you select the "No Scope" Mod because you can just hold down the Smoke button and "see" your cursor that way.

https://user-images.githubusercontent.com/115677736/195484084-ad887e57-d253-4c6c-b3f8-1f292513498f.mp4

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/20767
